v0.7/medical_imaging/3d-unet/Makefile

```diff
@@ -108,6 +116,7 @@ build_docker:
 
 .PHONY: launch_docker
 launch_docker: check_download_data_dir
+	@mkdir -p build/postprocessed_data
```

Comment:
Can we do `@mkdir -p $(PREPROCESSED_DATA_DIR)` just to keep the same style? Thanks

Reply:
Done.